### PR TITLE
Reviewing Lines Part 4

### PR DESCRIPTION
**Pull request by ttgmichael**
Dolar signs on QUESTLV100 lines 1300-1600 with some revisions.

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1298,114 +1298,114 @@ QUEST_LV_0100_20150317_001297	We are about to leave here, what's the matter?{nl}
...
+QUEST_LV_0100_20150317_001349	$I never thought that I would be able to see these beautiful patterns.{nl}When you go to Garden of Pillar, there may be more beautiful patterns. Come with me.
+QUEST_LV_0100_20150317_001350	$It seems that the Wandering Spirit wants to give the reward to someone in Fedimian.
+QUEST_LV_0100_20150317_001351	$After I finished the work, the Unicorn appeared.
 QUEST_LV_0100_20150317_001352	{memo X}Thank you for the regards from Laimonas. {nl} Come on in to Klaipeda. Uska is waiting.
-QUEST_LV_0100_20150317_001353	Don't forget to send my regards to the Klaipeda Chief Commander.
-QUEST_LV_0100_20150317_001354	When you are ready, please go to Aras in the Eastern Woods.
-QUEST_LV_0100_20150317_001355	Welcome! {nl} Oh, you're the Revelator. I really wanted to meet you. {nl}
-QUEST_LV_0100_20150317_001356	This is a Warp Scroll. {nl} You can go to any Goddess Statue or return to previous location by using it. {nl}
+QUEST_LV_0100_20150317_001353	$Don't forget to send my regards to the Klaipeda Knight Commander.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1353]: Don't forget to send my regards to the Knight Commander of Klaipeda.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
...
+QUEST_LV_0100_20150317_001420	$Four years ago, it was on the eve of the so called 'Medzio Diena'. {nl}Suddenly, a huge tree emerged in the middle of the capital and the earth and sky turned {nl}upside down. {nl}
+QUEST_LV_0100_20150317_001421	$I never knew a tree could grow that big. {nl}And the destruction of the capital began. {nl}
+QUEST_LV_0100_20150317_001422	$Everything happened so fast, and nobody knew what to do. {nl}We didn't know whether to run away or try to stop it... Our minds just became blank.
+QUEST_LV_0100_20150317_001423	$The destruction of the capital was just the beginning. {nl}Medzio Diena was simply a prelude to the catastrophes coming ahead. {nl}
+QUEST_LV_0100_20150317_001424	$Monstrous plants sprouted up all over and began devouring people.{nl}Everyone started running, but before I knew it I was the only person left.{nl}
+QUEST_LV_0100_20150317_001425	$Enough. Don't ask any more questions. {nl}I... don't want to think about it anymore.
+QUEST_LV_0100_20150317_001426	$The Excavation Camp on Zachariel's Royal Mausoleum is past here.{nl}You can freely enter, but please be careful not to disturb the people doing research.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Excavation Camp capitalized?

Comment (Sunaries) [inline QUEST_LV_0100.tsv:1426]: Did you change replace "Research Camp" with "Excavation Camp" in every single file?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
...
+QUEST_LV_0100_20150317_001430	$The Royal Mausoleum should be protected.{nl}If it can't be protected, it will be destroyed itself.
+QUEST_LV_0100_20150317_001431	$Fedimian is a very old city, and is surrounded by countless ruins and wonders. {nl}This includes the Mage Tower, the Cursed City and the Great Cathedral.
+QUEST_LV_0100_20150317_001432	$Hundreds of years ago an archmage by the name of Agailla Flurry built that tower. {nl}Much of the magic used today saw their beginnings within that tower, and haven't changed {nl}much since.
+QUEST_LV_0100_20150317_001433	$That city was abandoned after it was cursed with a spell that turned everything into stone. {nl}It is just beyond Pilgrim's Road.
+QUEST_LV_0100_20150317_001434	$It used to be the place where the people who supported the former rebel Ruklys lived. {nl}The curse is still in effect today so it's difficult to travel there.
+QUEST_LV_0100_20150317_001435	$Goddess Zemyna has always protected Jofuir for me. {nl}I believe my older brother also leaves me to Zemyna's care.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1435]: Goddess Zemyna has always protected us.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
...
-QUEST_LV_0100_20150317_001455	Where is he. I am scared.
+QUEST_LV_0100_20150317_001454	$The colleague who I used to work with has disappeared.{nl}He was definitely with me at the Forest of Fireflies, but I won't go back there. It's scary.
+QUEST_LV_0100_20150317_001455	$Where is he. I am scared.
 QUEST_LV_0100_20150317_001456	{memo X}My colleague is dead?{nl}That's nonsense. If I had not tempted him..{nl}
 QUEST_LV_0100_20150317_001457	{memo X}This map is useless now since my colleague is dead.{nl}Please pray that my colleague would go to the goddess safely.
-QUEST_LV_0100_20150317_001458	I'm screwed. Monsters took my bag. {nl}The bag contained everything I recorded on my journey.. What should I do?{nl}
-QUEST_LV_0100_20150317_001459	The goddess has surely sent you to help!{nl}Those monsters went down from here.
-QUEST_LV_0100_20150317_001460	I still have lots of things to do..
+QUEST_LV_0100_20150317_001458	$I'm screwed. Monsters took my bag. {nl}The bag contained everything I recorded on my journey.. What should I do?{nl}
+QUEST_LV_0100_20150317_001459	$The Goddess has surely sent help!{nl}Those monsters were last seen south of here.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1459]: Why did you capitalize goddess?

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
...
+QUEST_LV_0100_20150317_001473	$Monument
 QUEST_LV_0100_20150317_001474	$We have all become like wind in the canyon.{nl}Although we have left, our times together will always remain with you.
-QUEST_LV_0100_20150317_001475	Architect Laudi
-QUEST_LV_0100_20150317_001476	$I received a request to rebuild this structure, but after Medzio Diena, I have been stuck here.{nl}I don't know if the original client is still alive or if we lack the materials,{nl} but I will finish rebuilding this structure no matter what happens.
+QUEST_LV_0100_20150317_001475	$Architect Laudi
+QUEST_LV_0100_20150317_001476	$I received a request to rebuild this structure, but after Medzio Diena, I have been stuck here {nl}since. I don't know if the original client is still alive or if we lack the materials, but I will finish {nl}rebuilding this structure no matter what happens.
 QUEST_LV_0100_20150317_001477	{memo X}I will find the key to open the last seal.{nl}After opening a space with this scroll.
-QUEST_LV_0100_20150317_001478	The space may exist anywhere, but it may also not exist anywhere.
+QUEST_LV_0100_20150317_001478	$The space may exist anywhere, but it might also not exist anywhere.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1478]: The -> This?

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1426]: Pretty sure I did :P

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1435]: that's not what the Korean text says?

File: QUEST_LV_0100.tsv (hunk 5 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1478]: I don't think the line refers to a specific space.

And its actually for the spatial chasms so I'll change it to be more precise.

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1435]: 'Jofuir' is gibberish.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**